.gitignore

```diff
@@ -167,3 +167,5 @@ dist
 
 # Locally generated PII reports
 pii_report
+**/themes/
+**/themes/**/
```

Comment:
is this required?

Reply:
Yes suggested by dev

Reply:
please remove this, we might need to update in future